extensions/raindrop/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "raindrop-io",
+  "title": "Raindrop.io",
+  "version": "1.0.0",
+  "description": "Search your Raindrop.io bookmarks",
+  "icon": "raindrop-icon-big.png",
+  "author": "lardissone",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "latest_bookmarks",
+      "title": "Latest bookmarks",
+      "subtitle": "Raindrop.io",
+      "description": "Display your latest bookmarks",
+      "mode": "view"
+    },
+    {
+      "name": "search",
+      "title": "Search bookmarks",
+      "subtitle": "Raindrop.io",
+      "description": "Search bookmarks using operators",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "token",
+      "type": "password",
+      "required": true,
+      "title": "Access Token",
+      "description": "Your Raindrop.io Test Access Token.",
+      "link": "https://developer.raindrop.io/v1/authentication/token",
+      "placeholder": "Enter your Randrop.io Test Access token."
+    }
+  ],
+  "dependencies": {
+    "@raycast/api": "1.25.0",
+    "dayjs": "^1.10.7",
```

Comment:
I got an error trying to do `npm install` for this one.
>💥 Error: Build failed: Could not resolve "dayjs" (mark it as external to exclude it from the bundle)

Can you please take a look at this one? I can also propose to try [date fns](http://date-fns.org) in the case you won't be able to fix this.